packages/sdk/src/Sdk.ts

```diff
@@ -213,27 +216,24 @@ export class Sdk {
    * This methods serves to pre-fetch and loads only partially the financials.
    * It's recommended to call afterwards "getLlamaFinancials()".
    *
+   * @param chainId - ID of the chain
    * @param account - {@link Address} for the user
-   * @param chainType - for type of chains: mainnet or testnet
    */
   async getBorrowingVaultsFinancials(
-    account?: Address,
-    chainType: ChainType = ChainType.MAINNET
-  ): Promise<VaultWithFinancials[]> {
-    const res: VaultWithFinancials[] = [];
-    const chains = Object.values(CHAIN)
-      .filter((c) => c.chainType === chainType && c.isDeployed)
-      .map((c) => c.setConnection(this._configParams));
-
-    for (const chain of chains) {
-      const chainId = chain.chainId;
-      const vaults = VAULT_LIST[chainId].map((v) =>
-        v.setConnection(this._configParams)
-      );
-      const v = await batchLoad(vaults, account, chain);
-      res.push(...v);
+    chainId: ChainId,
+    account?: Address
+  ): Promise<FujiResult<VaultWithFinancials[]>> {
```

Comment:
Can we have a wrapper `FujiResultPromise<>`?

Reply:
After creating the wrapper `export type FujiResultPromise<T> = Promise<FujiResult<T>>`, I can't use it from the frontend since I get the following error

```Type 'FujiResultPromise' is not a valid async function return type in ES5/ES3 because it does not refer to a Promise-compatible constructor value.ts(1055)```

It works when calling the `sdk`, but we sometimes want to return the same type from the app, and that fails with the above error. Thoughts @brozorec ?

Reply:
Found [this](https://github.com/microsoft/TypeScript/issues/26781). Can you try changing target in tsconfig to "es6", tho Idk what other consequences it can have.

Reply:
That was it. I thought everything would break, that's why I didn't even consider changing the target, but did it after fixing solving the other comment and it worked smoothly without any issues (so far), so I'll push.